src/main/scala/org/broadinstitute/dsde/rawls/webservice/UserApiService.scala

```diff
@@ -24,6 +22,15 @@ trait UserApiService extends HttpService with PerRequestCreator with UserInfoDir
   val userServiceConstructor: UserInfo => UserService
 
   val userRoutes = requireUserInfo() { userInfo =>
+    path("user") {
+      post {
+        entity(as[RawlsUser]) { userToCreate =>
```

Comment:
this end point should not take any post body parameters. All information should come from OIDC headers as the post body is not trusted.